proxy-node-gateway/src/test/java/uk/gov/ida/notification/resources/EidasAuthnRequestResourceTest.java

```diff
@@ -116,21 +122,14 @@ private void setupHappyPath() throws URISyntaxException {
     }
 
     private void verifyHappyPath() {
-        GatewaySessionData expectedSessionData = new GatewaySessionData(
-            eidasSamlParserResponse,
-            vspResponse,
-            "eidas relay state"
-        );
-
-        verify(session).setAttribute(eq(SESSION_KEY_SESSION_DATA), captorGatewaySessionData.capture());
-        verify(session, times(3)).getId();
+        verify(sessionCacheManager, times(1)).addSession(eq("some session id"), any(GatewaySessionData.class));
```

Comment:
Tiny: shouldn't need the times(1) here, its implied as once, number of times used can then be verified with verifyNoMoreInteractions(..)